### DISCUSSION TITLE
How do you change the "home" directory from "~" to 🏠

### DISCUSSION THREAD

**Discussion by trajano** [starship/starship#1874] [Q&A] [answered]
How do you change the "home" directory from "~" to 🏠 (house emoji)

Also more generally how do we set it so some paths like 

~/Desktop get rendered as 💻 (desktop)
/ get rendered as 🖥 (computer)
~/Documents 📃

So I can get ~/My/Files be rendered as `🏠/My/Files`

**Comment by mickimnet** [comment, marked as answer]
You can take a look at my [configuration files](https://github.com/mickimnet/myth-prompt-themes), where I have done that extensively, and in order to achieve what you have asked for, you need to add the following to your `starship.toml` config file:

```toml
[directory.substitutions]
"~/Documents" = "📃"
"~/Desktop" = "💻"
"~" = "🏠"
```

I'm not sure if it is possible to substitute "/*", as I'm replacing the slash with a powerline symbol (see above).

Take care
Mick
